### PR TITLE
Streams Runner v1.1 Changes

### PR DESCRIPTION
**Pull request by pgerv12**
See generated pages here: https://pgerv12.github.io/streamsx.documentation/docs/beamrunner/beamrunner-1-intro/

For comparison: http://ibmstreams.github.io/streamsx.documentation/docs/beamrunner/beamrunner-1-intro/

### CODE REVIEW

File: docs/beamrunner/beamrunner-2a-using.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Because the application is launched on a remote system, the Streams job must be
 aware of your Beam application. To include your application and any dependencies,
 use the `--jarsToStage` option. For more information about this option, see [Streams Runner pipeline options](../beamrunner-5-ref/#streams-runner-pipeline-options).
 
-**Note**: Fat or uber JAR files can reduce the number of JAR files to stage. However, using them increases the size of the archive and affects upload and build times.
+**Note**: Fat or uber JAR files can reduce the number of JAR files to stage. Be mindful to not include dependencies already provided by the Streams Runner runner. Including redundant dependencies can increase the application archive size affecting upload and build times.
```
Comment (pamlane) [inline docs/beamrunner/beamrunner-2a-using.md]: Change Be mindful to not include dependencies already provided by the Streams Runner runner. Including redundant dependencies can increase the application archive size affecting upload and build times.
to
Be mindful to not include dependencies that are already provided by Streams Runner. Including redundant dependencies can increase the application archive size and affect upload and build times.

File: docs/beamrunner/beamrunner-5-ref.md [hunk 2]
```diff
@@ -55,7 +56,8 @@ The Streams Runner package contains the following folders:
     </tr>
     <tr>
       <td><code class="highlighter-rouge">jarsToStage</code></td>
-      <td>A list of JAR files (separated by colons) that are required to run the Apache Beam application. Include the JAR files that contain your program and any dependencies. (You don’t need to include Beam Google IO SDK or core Beam JAR files.) The listed JAR files are added to the SAB file.<br /><br /><strong>Note</strong>: The use of fat or uber JAR files increases the size of the SAB file and can negatively impact submission times to IBM Cloud.</td>
+      <td>A list of JAR files (separated by colons) that are required to run the Apache Beam application. Include the JAR files that contain your program and any dependencies. (You don’t need to include Beam Google IO SDK or core Beam JAR files.) The listed JAR files are added to the SAB file.<br /><br /><strong>Note</strong>: The use of fat or uber JAR files can reduce the number of JARs needed to be specified, but care should be taken not to include redundant JARs provided by the Streams
+      Runner. Including redundant dependencies can increase the application archive and can negatively impact submission times to IBM Cloud.</td>
```
Comment (mrshenli) [inline docs/beamrunner/beamrunner-5-ref.md]: Is this line break intentional?

File: docs/beamrunner/beamrunner-5-ref.md [hunk 3]
```diff
@@ -55,7 +56,8 @@ The Streams Runner package contains the following folders:
     </tr>
     <tr>
       <td><code class="highlighter-rouge">jarsToStage</code></td>
-      <td>A list of JAR files (separated by colons) that are required to run the Apache Beam application. Include the JAR files that contain your program and any dependencies. (You don’t need to include Beam Google IO SDK or core Beam JAR files.) The listed JAR files are added to the SAB file.<br /><br /><strong>Note</strong>: The use of fat or uber JAR files increases the size of the SAB file and can negatively impact submission times to IBM Cloud.</td>
+      <td>A list of JAR files (separated by colons) that are required to run the Apache Beam application. Include the JAR files that contain your program and any dependencies. (You don’t need to include Beam Google IO SDK or core Beam JAR files.) The listed JAR files are added to the SAB file.<br /><br /><strong>Note</strong>: The use of fat or uber JAR files can reduce the number of JARs needed to be specified, but care should be taken not to include redundant JARs provided by the Streams
```
Comment (pamlane) [inline docs/beamrunner/beamrunner-5-ref.md]: Change
The use of fat or uber JAR files can reduce the number of JARs needed to be specified, but care should be taken not to include redundant JARs provided by the Streams...
to
The use of fat or uber JAR files can reduce the number of JAR files that must be specified, but take care not to include JAR files that are provided by the Streams...

File: docs/beamrunner/beamrunner-5-ref.md [hunk 4]
```diff
@@ -81,23 +83,68 @@ The Streams Runner package contains the following folders:
...
 | Parameter | Description | Default value |
 | --- | --- | --- |
 | `restUrl` | The URL for the REST API when you use the `DISTRIBUTED` context type. This parameter is required if you implement metrics in the application. | [null] |
 | `userName` | The user name for basic authentication with REST API when you use the `DISTRIBUTED` context type. | [null] |
 | `userPassword` | The user password for basic authentication for REST API when you use the `DISTRIBUTED` context type. | [null] |
 
-<br>
-For the full list of pipeline options, enter  `--help=StreamsPipelineOptions` on the Beam application command line.
+## Environment Variables
+These environment variables are not required for the Streams Runner to work; however, they can be used for convenience when launching your Beam application.
```
Comment (pamlane) [inline docs/beamrunner/beamrunner-5-ref.md]: Change
however, they can be used for convenience when launching your Beam application.
to
however, they can be used for convenience when you launch your Beam application.

**Review by mrshenli — APPROVED** [review]
Thanks for updating the docs

File: docs/beamrunner/beamrunner-6-issues.md [hunk 5]
```diff
@@ -19,6 +19,11 @@ Learn about limitations and known issues for IBM® Streams Runner for Apache Bea
 - Experimental Splittable ParDos are not supported.
 
 ## Known issues
+### Issue
+Submission-time parameters specified while manually submitting SAB file (created via `BUNDLE` context) are not used during application runtime.
```
Comment (pamlane) [inline docs/beamrunner/beamrunner-6-issues.md]: Change to:
Submission-time parameters that are specified when you manually submit an SAB file (created by using the `BUNDLE` context) are not used during the application runtime.

File: docs/beamrunner/beamrunner-5-ref.md (hunk 2 above)
Comment (pgerv12) [reply docs/beamrunner/beamrunner-5-ref.md]: Nope! Good catch

**Pull request closed without merging**